### ISSUE TITLE
[bugfix-2.0.X] no z raise after G29 J

### ISSUE THREAD

**Issue by AcHub** [MarlinFirmware/Marlin#11077]
After ```G29 J```  is done with probing the bed, my nozzle doesn't raise before it moves. I have set ```#define Z_AFTER_PROBING 5``` but it doesn't seem to bother. Is it not used by G29?

**Comment by Roxy-3D** [comment]
I have several questions:
- Is this a problem with both G29 J 4 and G29 J  (with no parameter so it does 3 point leveling) ?  Which one are you doing?
- Do you see the nozzle lift the amount specified by  #define Z_CLEARANCE_BETWEEN_PROBES  at the end?

It looks like some of the probing logic has been changed.   And the G29 J stuff does not reflect that.   I'm going to zap both the bugfix_2.0.0 and bugfix_1.1.x branch to have this at the end of the G29 J routine:
```cpp
      STOW_PROBE();
      #ifdef Z_AFTER_PROBING
        move_z_after_probing();
      #endif
```
Can you please give the new version a try and see if it satisfies your expectations?

**Comment by AcHub** [comment]
It is only an issue with G29 J (3 Points). When setting G29 J 2 or G29 J 4, the nozzle is raised before the last move to the center.
With G29 J, the nozzle is not raised at all at the end of probing, before it moves to the center of the bed. It only raises between the probing points.

I just tested with your latest commit and now it raises at the end of G29 J, so this is a success, thanks.
Actually it seems like it is even raising twice in a row now. But I can't tell if the first time is the distance set as ```Z_CLEARANCE_BETWEEN_PROBES``` or ```Z_CLEARANCE_DEPLOY_PROBE``` (both are 2mm) and on top of that the raise after probing, as it moves too fast to identify. 
All I can say is that the nozzle finally rests 5mm over the printbed, which is what I had set as ```Z_AFTER_PROBING```
